docs/Displaying a User Interface/Alerts and Subtle Alerts/index.md

```diff
@@ -501,4 +501,428 @@ if (response.getSuccess()) {
+```
+!@
+
+@![android,javaSE,javaEE]
+```java
+subtleAlert.setTtsChunks(Collections.singletonList(new TTSChunk("Text to Speak", SpeechCapabilities.TEXT)));
+```
+!@
+
+@![javascript]
+```js
+const chunk = new SDL.rpc.structs.TTSChunk()
+    .setType(SDL.rpc.enums.SpeechCapabilities.TEXT)
+    .setText('Text to Speak');
+subtleAlert.setTtsChunks([chunk]);
+```
+!@
+
+#### Sound File
+The `ttsChunks` parameter can also take a file to play/speak. For more information on how to upload the file please refer to the [Playing Audio Indications](Speech and Audio/Playing Audio Indications) guide.
+
+@![iOS]
+##### Objective-C
+```objc
+subtleAlert.ttsChunks = [SDLTTSChunk fileChunksWithName:@"<#Name#>"];
+```
+
+##### Swift
+```swift
+subtleAlert.ttsChunks = SDLTTSChunk.fileChunks(withName: "<#Name#>")
+```
+!@
+
+@![android,javaSE,javaEE]
+```java
+TTSChunk ttsChunk = new TTSChunk(sdlFile.getName(), SpeechCapabilities.FILE);
+subtleAlert.setTtsChunks(Collections.singletonList(ttsChunk));
+```
+!@
+
+@![javascript]
+```js
+const ttsChunk = new SDL.rpc.structs.TTSChunk()
+    .setText(sdlFile.getName())
+    .setType(SDL.rpc.enums.SpeechCapabilities.FILE);
+subtleAlert.setTtsChunk([ttsChunk]);
+```
+!@
+
+## Showing the Subtle Alert
+
+@![iOS]
+##### Objective-C
+```objc
+[self.sdlManager sendRequest:subtleAlert withResponseHandler:^(SDLRPCRequest *request, SDLRPCResponse *response, NSError *error) {
+    if (!response.success.boolValue) { 
+        // Print out the error if there is one and return early
+        return;
+    }
+    <#Subtle Alert was shown successfully#>
+}];
+```
+
+##### Swift
+```swift
+sdlManager.send(request: subtleAlert) { (request, response, error) in
+    guard response?.success.boolValue == true else { return }
+    <#Subtle Alert was shown successfully#>
+}
+```
+!@
+
+@![android,javaSE,javaEE]
+```java
+// Handle RPC response
+subtleAlert.setOnRPCResponseListener(new OnRPCResponseListener() {
+    @Override
+    public void onResponse(int correlationId, RPCResponse response) {
+      if (response.getSuccess()){
+        DebugTool.logInfo(TAG, "Subtle Alert was shown successfully");
+      }
+    }
+});
+sdlManager.sendRPC(subtleAlert);
+```
+!@
+
+@![javascript]
+```js
+const response = await sdlManager.sendRpcResolve(subtleAlert);
+if (response.getSuccess()) {
+    console.log('Subtle alert was shown successfully');
+}
+// thrown exceptions should be caught by a parent function via .catch()
+
+if (response.getSuccess()) {
+    console.log('Subtle alert was shown successfully');
+}
+```
+!@
+
+## Dismissing the Subtle Alert
+You can dismiss a displayed subtle alert before the timeout has elapsed.
+
+Please note that canceling the subtle alert will only dismiss the displayed alert. If you have set the `ttsChunk` property, the speech will play in its entirety even when the displayed subtle alert has been dismissed. If you know you will cancel a subtle alert, consider setting a short `ttsChunk`.
```

Comment:
```suggestion
!!! NOTE
Canceling the subtle alert will only dismiss the displayed alert. If you have set the `ttsChunk` property, the speech will play in its entirety even when the displayed subtle alert has been dismissed. If you know you will cancel a subtle alert, consider setting a short `ttsChunk`.
!!!
```

Reply:
Do you think this would look better?

Reply:
I think this change makes sense should we also do the same for the Alert section